src/block/horizontal-scroller/frontend-horizontal-scroller.js

```diff
@@ -22,6 +30,10 @@ class StackableHorizontalScroller {
 
 				// Prevent selection of contents because of dragging.
 				e.preventDefault()
+				// adds the on click event listener to each inner column
+				children.forEach( child => {
+					child.addEventListener( 'click', onClickHandler )
+				 } )
```

Comment:
I'd like to suggest that for commenting this one, while we do add the click event listener here, this is already apparent in the code. It would be more helpful for other people to know why we are adding a click event listener in this step. I'd probably write something like this:

```suggestion
				// Links will trigger while dragging, this disabled the links.
				children.forEach( child => {
					child.addEventListener( 'click', onClickHandler )
				 } )
```